dnf-plugins-core.spec

```diff
@@ -42,6 +42,7 @@ Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
 BuildArch:      noarch
 BuildRequires:  cmake
 BuildRequires:  gettext
+Requires:       preproc-rpmspec >= 1.2
```

Comment:
This cannot be a mandatory dependency, as it will not be pulled into RHEL. Additionally, other distributions will not necessarily have this tool packaged either.

Please rework so that this is an optional dependency.

Reply:
Thanks for the review. I did the changes. Please, take a look.